code/datums/status_effects/debuffs.dm

```diff
@@ -971,11 +971,13 @@
 		for(var/organ_slot in organs_to_damage)
 			if(prob(75))
 				carbon_owner.adjustOrganLoss(organ_slot, 20)
-
+		var/list/item_blacklist = list(/obj/item/grenade)
 		// And roughly 75% of their items will take a smack, too
 		for(var/obj/item/thing in carbon_owner.get_all_gear())
 			if(!QDELETED(thing) && prob(75))
-				thing.take_damage(100)
+				for(var/blacklisted in item_blacklist)
+					if(!istype(thing, blacklisted))
+						thing.take_damage(100)
```

Comment:
It'd be better if you made this a static variable, and made it use the ``typecacheof`` approach. You can see an example, although poor, of how this works in ``carp_lasso.dm``.